elftools/elf/descriptions.py

```diff
@@ -35,7 +35,13 @@ def describe_ei_osabi(x):
     return _DESCR_EI_OSABI.get(x, _unknown)
 
 
-def describe_e_type(x):
+def describe_e_type(x, elf=None):
```

Comment:
Nit: this parameter is called `elffile` in other `describe_*` functions that use it. Please change for consistency.